src/store/store.ts

```diff
@@ -0,0 +1,109 @@
+import { configureStore, combineReducers, Reducer } from "@reduxjs/toolkit";
```

Comment:
[note] A lot of this `src/store/store.ts` file was just refactored here from `src/index.tsx` + updated to use the syntax/helpers/etc that come with `@reduxjs/toolkit`